DataFormats/Detectors/PHOS/include/DataFormatsPHOS/Cluster.h

```diff
@@ -82,7 +78,7 @@ class Cluster
   float mTime = 0.;              ///< Time of the digit with maximal energy deposition
   float mDistToBadChannel = 999; ///< Distance to nearest bad crystal
 
-  ClassDefNV(Cluster, 1);
+  ClassDefNV(Cluster, 2);
```

Comment:
It does not harm, but the class version need to be changed if the class layout was modified. Modifying some methods does not require this.

Reply:
I removed datamember: ref to MCLabel. Normally one should increase version in this case. But as we not not have a lot of data, we can keep 1.

Reply:
Sorry, missed this, the increment is indeed needed (and does not harm anyway).